src/vivarium_census_prl_synth_pop/components/person.py

```diff
@@ -192,6 +197,62 @@ def _perform_new_household_moves(
 
         return pop
 
+    def _perform_gq_person_moves(self, pop: pd.DataFrame, movers: pd.Index) -> pd.DataFrame:
+        """
+        Move each simulant in movers to a random GQ type.
+        """
+        if len(movers) == 0:
+            return pop
+
+        # The two GQ housing type categories (institutional, non-institutional) are
+        # tracked in the "relation_to_household_head" column, even though
+        # that column name doesn't really make sense in a GQ setting.
+        categories = list(data_values.GROUP_QUARTER_IDS.keys())
+        gq_address_ids = (
+            pop[pop["relation_to_household_head"].isin(categories)][
+                ["household_id", "address_id"]
+            ]
+            .drop_duplicates()
+            .set_index("household_id")["address_id"]
+        )
+        assert gq_address_ids.index.is_unique
+
+        housing_type_category_values = self.randomness.choice(
```

Comment:
Doesn't this move them with equal probability to institutional or non-institutional GQ? We have a bunch of data for moving into GQ that I don't see used here. Has something changed in the model?